--- love-0.9.1/src/Makefile.in	2014-04-01 14:26:09.000000000 -0400
+++ ../dev/love-0.9.1/src/Makefile.in	2014-06-19 02:40:32.724874733 -0400
@@ -2719,7 +2719,7 @@
 	modules/sound/lullaby/$(am__dirstamp) \
 	modules/sound/lullaby/$(DEPDIR)/$(am__dirstamp)
 liblove.la: $(liblove_la_OBJECTS) $(liblove_la_DEPENDENCIES) $(EXTRA_liblove_la_DEPENDENCIES) 
-	$(AM_V_CXXLD)$(liblove_la_LINK) -rpath $(libdir) $(liblove_la_OBJECTS) $(liblove_la_LIBADD) $(LIBS)
+	$(AM_V_CXXLD)$(liblove_la_LINK) $(liblove_la_OBJECTS) $(liblove_la_LIBADD) $(LIBS)
 install-binPROGRAMS: $(bin_PROGRAMS)
 	@$(NORMAL_INSTALL)
 	@list='$(bin_PROGRAMS)'; test -n "$(bindir)" || list=; \
