diff -ur devil-1.7.8/src-IL/src/il_icon.c ../dev/devil-1.7.8/src-IL/src/il_icon.c
--- devil-1.7.8/src-IL/src/il_icon.c	2009-03-08 03:10:09.000000000 -0400
+++ ../dev/devil-1.7.8/src-IL/src/il_icon.c	2014-06-27 15:20:10.587123491 -0400
@@ -525,7 +525,7 @@
 
 	// Expand low-bit-depth grayscale images to 8 bits
 	if (ico_color_type == PNG_COLOR_TYPE_GRAY && bit_depth < 8) {
-		png_set_gray_1_2_4_to_8(ico_png_ptr);
+		png_set_expand_gray_1_2_4_to_8(ico_png_ptr);
 	}
 
 	// Expand RGB images with transparency to full alpha channels
diff -ur devil-1.7.8/src-IL/src/il_png.c ../dev/devil-1.7.8/src-IL/src/il_png.c
--- devil-1.7.8/src-IL/src/il_png.c	2009-03-08 03:10:09.000000000 -0400
+++ ../dev/devil-1.7.8/src-IL/src/il_png.c	2014-06-27 15:20:05.878991933 -0400
@@ -278,7 +278,7 @@
 
 	// Expand low-bit-depth grayscale images to 8 bits
 	if (png_color_type == PNG_COLOR_TYPE_GRAY && bit_depth < 8) {
-		png_set_gray_1_2_4_to_8(png_ptr);
+		png_set_expand_gray_1_2_4_to_8(png_ptr);
 	}
 
 	// Expand RGB images with transparency to full alpha channels
