--- LuaJIT-2.0.3/Makefile	2014-03-12 08:10:00.000000000 -0400
+++ ../dev/LuaJIT-2.0.3/Makefile	2014-06-12 23:03:00.246391387 -0400
@@ -24,7 +24,9 @@
 # Change the installation path as needed. This automatically adjusts
 # the paths in src/luaconf.h, too. Note: PREFIX must be an absolute path!
 #
-export PREFIX= /usr/local
+# #UNCONDITIONAL#LOVE# reminds you to not modify 'export PREFIX', which
+# is set automagically by ./setup.sh configure
+export PREFIX= INVALID
 export MULTILIB= lib
 ##############################################################################
 
